Assets/Scripts/Interactables/Dialogue/DialogueInteractable.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DialogueInteractable : Interactable
+{
+    public Conversation conversation;
+
+    private GameObject NPCBubble;
+    private GameObject BirdieBubble;
+    private SpeakerUI speakerUILeft;
+    private SpeakerUI speakerUIRight;
+
+    private int activeLineIndex = 0;
+
+    public override void OnInteract(GameObject birdie)
+    {
+        BirdieBubble = birdie.transform.GetChild(1).gameObject; 
+        NPCBubble = this.transform.GetChild(1).gameObject;

```

Comment:
We should figure out what's the best practice for this too

Reply:
Aside from my comment below, I'd keep in mind the naming convention of regular variables starting with a lowercase letter!

Reply:
Yeah! This one is risqué! so the user needs to make sure they drag the speech bubble as the second child of the object, i have it in the instructions!